src/main/java/org/jabref/gui/fieldeditors/DateEditorViewModel.java

```diff
@@ -36,7 +36,7 @@ public TemporalAccessor fromString(String string) {
                 if (StringUtil.isNotBlank(string)) {
                     try {
                         return dateFormatter.parse(string);
-                    } catch (DateTimeParseException exception) {
+                    } catch (Exception exception) {
```

Comment:
Why did you change this to a generic exception? Exception should be always the most specfic one